src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -95,12 +96,12 @@ public function __construct(OutputInterface $output, int $max = 0, float $minSec
     }
 
     /**
-     * Sets a placeholder formatter for a given name.
+     * Sets a placeholder formatter for a given name, globally for all instances of ProgressBar
      *
      * This method also allow you to override an existing placeholder.
      *
      * @param string   $name     The placeholder name (including the delimiter char like %)
-     * @param callable $callable A PHP callable
+     * @param callable(ProgressBar):string $callable A PHP callable
      */
     public static function setPlaceholderFormatterDefinition(string $name, callable $callable): void
```

Comment:
If someone wonders, as I do, if this static method should be depreciated: then I think not because it is very widely used.
https://github.com/search?l=PHP&q=setPlaceholderFormatterDefinition&type=Code